parent/pom.xml

```diff
@@ -76,8 +76,7 @@
         <hamcrest.version>1.2</hamcrest.version>
         <httpclient.version>4.5.1</httpclient.version>
         <httpcore.version>4.4.3</httpcore.version>
-        <jackson.version>2.9.9</jackson.version>
-        <jackson1.version>1.9.13</jackson1.version>
```

Comment:
Please revert this. This changes doesn't related with your ticket

Reply:
Reverted